### DISCUSSION TITLE
Can i prerender routes in nuxt2 like nuxt3?

### DISCUSSION THREAD

**Discussion by nieyuyao** [nuxt/nuxt#20317] [Questions (Nuxt 2)] [answered]
I want prerender some routes in nuxt2. In nuxt3, this can be done by `routeRules`.

```javascript
routeRules: {
    '/static': { prerender: true },
 }
```
Is there a similar feature in nuxt2?

**Comment by ndragun92** [comment, marked as answer]
Nuxt 2 does not support hybrid rendering. You can have SSR/SPA or SSG as a whole thing not per page
